### PR TITLE
Feature: ssh directly into container

### LINKED ISSUE #140: ssh directly into docker image (as opposed to host)

### PR DESCRIPTION
**Pull request by jafreck**
Fix #140

### CODE REVIEW

File: aztk/config.py [hunk 1]
```diff
@@ -273,10 +274,13 @@ def _merge_dict(self, config):
         if 'jupyter_port' in config and config['jupyter_port'] is not None:
             self.jupyter_port = config['jupyter_port']
 
-        if 'connect' in config and config['connect'] is False:
-            self.connect = False
+        if 'host' in config and config['host'] is not None:
```
Comment (timotheeguerin) [inline aztk/config.py]: you can do `config.get('host') is not None` instead of this double check here

File: aztk/clusterlib.py [hunk 2]
```diff
@@ -359,7 +366,7 @@ def pretty_dedicated_node_count(self, cluster: ClusterModel)-> str:
             return '{}'.format(cluster.dedicated_nodes)
 
     def pretty_low_pri_node_count(self, cluster: ClusterModel)-> str:
-        if (cluster.pool.allocation_state is batch_models.AllocationState.resizing\
+        if (cluster.pool.allocation_state is batch_models.AllocationState.resizing
```
Comment (timotheeguerin) [inline aztk/clusterlib.py]: removed \ only on this line?

Comment (jafreck) [reply aztk/clusterlib.py]: I don't think it's necessary. It is done the same way in pretty_dedicated_node_count() just above this.

Comment (timotheeguerin) [reply aztk/clusterlib.py]: i mean there is one next line too that is not needed

**Pull request merged**